source/cli/appservices-function.txt

```diff
@@ -69,10 +69,12 @@ Inherited Options
 Related Commands
 ----------------
 
+* :ref:`appservices-function-list` - List the Functions of your app
```

Comment:
"Functions _of_ your app" reads weird to me
```suggestion
* :ref:`appservices-function-list` - List the Functions defined in your app
```
or maybe "defined _for_ your app"?